### PR TITLE
added init repo files

### PR DESCRIPTION
**Pull request by VlasenkoYevhenii**

### CODE REVIEW

**Review by ValentynSharshon — APPROVED** [review]
GJ!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
```
Comment (makaroshyna) [inline .github/workflows/ci.yml:14]: Maybe let's change the version to 21?

**Review by makaroshyna — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 17
```
Comment (makaroshyna) [inline .github/workflows/ci.yml:11]: Change this naming too.

**Review by makaroshyna — CHANGES_REQUESTED** [review]

**Review by makaroshyna — APPROVED** [review]
All good!

**Pull request merged**